projects/envoy/Dockerfile

```diff
@@ -27,12 +27,9 @@ RUN apt-get update && apt-get -y install  \
     libtool         \
     wget            \
     golang          \
-    python	    \
     rsync
 
-# Install Bazelisk
-RUN wget -O /usr/local/bin/bazel https://github.com/bazelbuild/bazelisk/releases/download/v0.0.8/bazelisk-linux-amd64
-RUN chmod +x /usr/local/bin/bazel
```

Comment:
thanks for this! just realized bazel is already installed through bazelisk.

Reply:
Yeah, it was actually added pretty recently (#4976), in preparation for the Bazel support :)